src/main/java/io/dropwizard/jdbi/unitofwork/core/JdbiUnitOfWorkProvider.java

```diff
@@ -0,0 +1,129 @@
+package io.dropwizard.jdbi.unitofwork.core;
+
+import com.google.common.collect.Sets;
+import com.google.common.reflect.Reflection;
+import org.reflections.Reflections;
+import org.reflections.scanners.Scanners;
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.sqlobject.SqlBatch;
+import org.skife.jdbi.v2.sqlobject.SqlCall;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@SuppressWarnings({"UnstableApiUsage", "rawtypes", "unchecked"})
+public class JdbiUnitOfWorkProvider {
```

Comment:
This is the only point of access for the users unless they want to do something low level for which a getter is exposed `getHandleManager()`